pkg/operator/credentialsrequest/status.go

```diff
@@ -82,6 +86,7 @@ func (r *ReconcileCredentialsRequest) syncOperatorStatus() error {
 	if !clusteroperator.ConditionsEqual(oldConditions, co.Status.Conditions) ||
 		!reflect.DeepEqual(oldVersions, co.Status.Versions) ||
 		!reflect.DeepEqual(oldRelatedObjects, co.Status.RelatedObjects) {
+		log.Infof("full obj: %v", co.Status.Conditions)
```

Comment:
Debugf() seems more appropriate

Reply:
Killed all these, they were for debugging a post-backport problem.